physx/src/scene.rs

```diff
@@ -503,6 +538,11 @@ impl SceneBuilder {
         self
     }
 
+    pub fn use_controller_manager(&mut self, use_controller_manager: bool) -> &mut Self {
```

Comment:
If we could also pass in the controller manager locking flag here that'd be lovely!